### PR TITLE
Add helper methods for adding multiple Migrations at once

### PR DESCRIPTION
**Pull request by gwynne**
- Adds an overload of `Migrations.add(_:to:)` which accepts an array of `Migration`, e.g.:
        
    ````swift
    app.migrations.add([ConfigModel.Migration(), SolarSystem.Migration()])
    app.migrations.add([UserModel.Migration(), PostModel.Migration()], to: .psql)
    ````

    As a new public API, this requires a `semver-minor` release.

- Also adds an overload which accepts a variadic parameter of `Migration` type:
        
    ````swift
    app.migrations.add(ConfigModel.Migration(), SolarSystem.Migration())
    app.migrations.add(UserModel.Migration(), PostModel.Migration(), to: .psql)
    ````

- Adds a benchmark test for the new methods, just for the heck of it.

### CODE REVIEW

File: Sources/FluentKit/Migration/Migrations.swift [hunk 1]
```diff
@@ -14,4 +14,12 @@ public final class Migrations {
     public func add(_ migration: Migration, to id: DatabaseID? = nil) {
         self.storage.append(.init(id: id, migration: migration))
     }
+
+    public func add(_ migrations: Migration..., to id: DatabaseID? = nil) {
```
Comment (MrLotU) [inline Sources/FluentKit/Migration/Migrations.swift:18]: Make this `@inlinable`?

File: Sources/FluentKit/Migration/Migrations.swift [hunk 2]
```diff
@@ -14,4 +14,12 @@ public final class Migrations {
...
+        self.add(migrations, to: id)
+    }
+
+    public func add(_ migrations: [Migration], to id: DatabaseID? = nil) {
```
Comment (MrLotU) [inline Sources/FluentKit/Migration/Migrations.swift:22]: If we make this call `add(_ migration:to:)` we could make this `@inlinable` too?

**Comment by MrLotU** [comment]
Also, propose to rename to `Add helper methods for adding multiple Migration's at once`

File: Sources/FluentKit/Migration/Migrations.swift (hunk 2 above)
Comment (gwynne) [reply Sources/FluentKit/Migration/Migrations.swift:22]: Calling `.append(contentsOf: .map { ... })` should be sufficiently more efficient than `.forEach { .append($0) }` to warrant not chaining here.

Comment (MrLotU) [reply Sources/FluentKit/Migration/Migrations.swift:22]: Ah, yeah that makes sense 👍

**Comment by gwynne** [comment]
CI failures are due to https://github.com/OpenKitten/MongoKitten/pull/245

**Comment by tanner0101** [comment]
These changes are now available in [1.0.0-rc.2.1](https://github.com/vapor/fluent-kit/releases/tag/1.0.0-rc.2.1)

**Pull request merged**